cni/network/network.go

```diff
@@ -589,6 +601,14 @@ func (plugin *netPlugin) Delete(args *cniSkel.CmdArgs) error {
 
 	log.Printf("[cni-net] Read network configuration %+v.", nwCfg)
 
+	plugin.report.OperationType = "DEL"
+	if nwCfg.MultiTenancy {
+		plugin.report.Context = "AzureCNIMultitenancy"
+	}
+	plugin.report.SubContext = fmt.Sprintf("%+v", nwCfg)
+	plugin.report.BridgeDetails.NetworkMode = nwCfg.Mode
+	plugin.report.InterfaceDetails.SecondaryCAUsedCount = plugin.nm.GetNumberOfEndpoints("", nwCfg.Name)
```

Comment:
could these lines be encapsulated in one function, seems like lots of repetitive code.